data/basics/userBalance.go

```diff
@@ -303,36 +307,38 @@ func MakeAccountData(status Status, algos MicroAlgos) AccountData {
 }
 
 // Money returns the amount of MicroAlgos associated with the user's account
-func (u AccountData) Money(proto config.ConsensusParams, rewardsLevel uint64) (money MicroAlgos, rewards MicroAlgos) {
+func (u *AccountData) Money(proto config.ConsensusParams, rewardsLevel uint64) (money MicroAlgos, rewards MicroAlgos) {
 	e := u.WithUpdatedRewards(proto, rewardsLevel)
 	return e.MicroAlgos, e.RewardedMicroAlgos
 }
 
 // WithUpdatedRewards returns an updated number of algos in an AccountData
 // to reflect rewards up to some rewards level.
-func (u AccountData) WithUpdatedRewards(proto config.ConsensusParams, rewardsLevel uint64) AccountData {
+func (u *AccountData) WithUpdatedRewards(proto config.ConsensusParams, rewardsLevel uint64) AccountData {
```

Comment:
Can you please remove the changes from `AccountData` to `*AccountData` from this PR? I don't want to change things if it won't make a performance difference.

Reply:
done